nRF Toolbox/WebViewController/WebViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WebViewController.swift
+//  nRF Toolbox
+//
+//  Created by Nick Kibysh on 27/08/2019.
+//  Copyright © 2019 Nordic Semiconductor. All rights reserved.
+//
+
+import UIKit
+
+class WebViewController: UIViewController {
+    var webView: UIWebView { return self.view as! UIWebView }
+    let link: LinkService
+    
+    override func loadView() {
+        self.view = UIWebView()
+    }
+    
+    init(link: LinkService) {
+        self.link = link
+        super.init(nibName: nil, bundle: nil)
+        let request = URLRequest(url: link.url)
+        self.webView.loadRequest(request)
+        
+        self.navigationItem.title = link.name
+        
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Open", style: .plain, target: self, action: #selector(openInSafari))
```

Comment:
I'd change the title, because "Open" is a bit difficult to understand - after all, the link is already "open".

My suggestion would be to use an "export" or "share" like icon instead to perform the action, or even pop up a context menu to allow the user to copy the link, open in Safari, etc.

Reply:
Hm, context menu is great solution.
Thank you!

Reply:
Ok, now it looks like this.
![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2019-08-28 at 11 01 01](https://user-images.githubusercontent.com/4790759/63848503-3bf69480-c990-11e9-88cb-010a72d4b515.png)
![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2019-08-28 at 11 16 44](https://user-images.githubusercontent.com/4790759/63848504-3bf69480-c990-11e9-89ae-f24b18ed34cb.png)
![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2019-08-28 at 11 16 47](https://user-images.githubusercontent.com/4790759/63848505-3c8f2b00-c990-11e9-8803-0ca0967bbfa6.png)